docs/architecture/README.md

```diff
@@ -1,3 +1,9 @@
+---
```

Comment:
ditto. can we avoid this being rendered in markdown?

Reply:
Please see comments above